.travis.yml

```diff
@@ -25,4 +25,4 @@ install:
 
 script:
   - make test
-  - [ -z "$COMPOSER_OPTS" ] && make package
```

Comment:
Travis was failing to interpret the manifest file and wasn't testing any new builds.